### ISSUE TITLE
itest error: Error invalid chain_id, should be 1020, yours:1024

### ISSUE THREAD

**Issue by jimoquinn** [iost-official/go-iost#1087]
iTest needs to have he chain_id updated in 3.1.0.  Running it gives the following error:

Error 2019-05-12 03:59:04.265 itest.go:116 Create account failed: rpc error: code = Unknown desc = VerifyError invalid chain_id, should be 1020, yours:1024

**Comment by ziranliu** [comment]
This problem has been fixed by this commit: https://github.com/iost-official/go-iost/commit/067f4ee5e78393fcace13f4acd68f09f429dadc2
Please pull the latest code and try again.

**Comment by joelcho** [comment]
i got same problem in 3.1.0
```
iwallet publish Token.js Token.abi --account admin
```
```
....
....
...
send tx error rpc error: code = Unknown desc = VerifyError invalid chain_id, should be 1020, yours:1024
```

i've try pull & compile the [latest commit](https://github.com/iost-official/go-iost/commit/2b191e03f17c06c8b6f7983d02ee1f68bee72d36) , but not work for me.

@ziranliu said fixed by [067f4ee](https://github.com/iost-official/go-iost/commit/067f4ee5e78393fcace13f4acd68f09f429dadc2), but why `itest` only ?

**Comment by jimoquinn** [comment]
Latest code solved the issue.
